game/src/ecs/components/AnimationComponent.java

```diff
@@ -10,8 +10,9 @@
 import semanticAnalysis.types.DSLTypeMember;
 
 /**
- * AnimationComponent is a component that stores the possible animations and the current animation
- * of an entity
+ * The AnimationComponent associates an entity with its animations. It stores the current animation
+ * and two idle animations, one for each direction (left/right). The current animation can be
+ * overwritten by using the {@link #setCurrentAnimation} of this component.
```

Comment:
Hier fehlt noch eine kurze Erklärung, wie so eine Animation aussieht/funktioniert: Es sind mehrere Bilder in einem gemeinsamen Ordner, die im Dateinamen durchnummeriert sind und die pro "Tick" (Frame?) durchiteriert werden und vom ??? für die Entität angezeigt werden.

Reply:
Ist das nicht eher relevant in der Dokumentation von `Animation` selbst? Das `AnimationComponent` verwendet ja schließlich nur die Animationen.

Reply:
haste wahr. trotzdem, ein kurzer hinweis mit javadoc-link auf das system schadet vermutlich nicht.